atomic_reactor/constants.py

```diff
@@ -14,6 +14,77 @@
 BUILD_JSON_ENV = 'BUILD_JSON'
 RESULTS_JSON = 'results.json'
 
+JSON_SCHEMA = {
```

Comment:
What object is this describing? It doesn't look like a Build, and it doesn't look like the ATOMIC_REACTOR_PLUGINS environment variable either.

FWIW I think it only makes sense to validate ATOMIC_REACTOR_PLUGINS, not the full Build JSON (which OpenShift will already have validated).